src/cordova_firebase.mli

```diff
@@ -32,23 +32,20 @@ val log_event : event:string ->
 val set_enabled : bool -> unit
   [@@js.global "cordova.plugins.firebase.analytics.setEnabled"]
 
-val get_app_instance_id : unit -> string
-  [@@js.global "cordova.plugins.firebase.analytics.getAppInstanceId"]
-
 [@@@js.stop]
 
-val available : unit -> bool
+val firebase_available : unit -> bool
```

Comment:
Maybe we could simply keep the name without the prefix `firebase`.

Reply:
I wanted to avoid confusion but it's probly juste repetition